src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -235,8 +236,9 @@ export class WizardModal extends Modal {
 	}
 
 	public open(): void {
-		this.showPage(0, false, true);
-		this.show();
+		this.showPage(0, false, true).then(() => {
```

Comment:
This isn't really changing any behavior as it is currently - the only async thing in the showPage function is the validation and we aren't doing that (second parameter) so it's going to be all synchronous anyways. But I figured it was better to do this since we want the showPage stuff to be finished before displaying the actual dialog otherwise it could result in things changing after it's displayed if the showPage is changed to have more async stuff.